world/achievement_mutation_manager.cpp

```
@@ -0,0 +1,430 @@
+				pending.request,
+				std::move(pending.character_ids),
+				true
+			);
+			continue;
+		}
+
+		Wake(pending.character_ids);
+	}
+}
+
+bool AchievementMutationManager::Persist(
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)